README.adoc

```diff
@@ -115,17 +92,22 @@ First step, you have to build the Docker image
 
 In order to run the build, you have to launch the following command:
 
-    $ docker run -v $PWD/.m2:/root/.m2 -v $PWD:/origin -v $PWD/dockerfiles/run/spring/destination:/destination -t james/project -s SHA1
+    $ docker run -v $PWD/.m2:/root/.m2 -v $PWD:/origin -t james/project -s SHA1
```

Comment:
I agree about the problem of adopting a new docker feature. Nonetheless, we should propose a strategy that doesn't involve copy/pasting a command line then understand that I should have mounted a volume and do it again.